### ISSUE TITLE
Assertion in postfilter.c with certain input data

### ISSUE THREAD

**Issue by KvanTTT** [freedv/codec2#6]
Can not decompress this data packet at 3200 bit/sec mode because of ```assert(e > 0.0)``` in postfilter.c:

**24 88 1 3 140 208 131**

How should I check input data in order to avoid this assertion? (My .NET app that wrapped codec2.dll terminated after this assertion).

**Comment by drowe67** [comment]
Pls send me a binary file that, when used as input to c2dec, reproduces 
the issue.

**Comment by KvanTTT** [comment]
Certainly: https://dl.dropboxusercontent.com/u/26130928/codec2%20postfilter%20assertion.bin

**Comment by drowe67** [comment]
OK try the latest codec-dev
